Examples/Physics_applications/free_electron_laser/analysis_fel.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+
+"""
+This script tests that the FEL is correctly modelled in the simulation.
+
+The physical parameters are the same as the ones from section 5.1
+of https://arxiv.org/pdf/2009.13645
+
+The simulation uses the boosted-frame technique as described in
+https://www.osti.gov/servlets/purl/940581
+In particular, the effect of space-charge is effectively turned off
+by initializing an electron and positron beam on top of each other,
+each having half the current of the physical beam.
+
+The script checks that the radiation wavelength and gain length
+are the expected ones. The check is performed both in the
+lab-frame diagnostics and boosted-frame diagnostics.
+"""
+
+import os
+import sys
+
+import numpy as np
+from openpmd_viewer import OpenPMDTimeSeries
+from scipy.constants import c, e, m_e
+
+sys.path.insert(1, "../../../../warpx/Regression/Checksum/")
+import checksumAPI
+
+# Physical parameters of the test
+gamma_bunch = 100.6
+Bu = 0.5
+lambda_u = 3e-2
+k_u = 2 * np.pi / lambda_u
+K = e * Bu / (m_e * c * k_u)  # Undulator parameter
+gamma_boost = (
+    gamma_bunch / (1 + K * K / 2) ** 0.5
+)  # Lorentz factor of the ponderomotive frame
+beta_boost = (1 - 1.0 / gamma_boost**2) ** 0.5
+
+
+# Analyze the diagnostics showing quantities in the lab frame
+ts_lab = OpenPMDTimeSeries("diags/diag_labframe")
```

Comment:
This could be read from the command line as well, right?
```suggestion
# Analyze the diagnostics showing quantities in the lab frame
filename = sys.argv[1]
ts_lab = OpenPMDTimeSeries(filename)
```